packages/mui-utils/src/useIsFocusVisible/useIsFocusVisible.test.js

```diff
@@ -85,7 +85,12 @@ describe('useIsFocusVisible', () => {
       document.body.removeChild(rootElement);
     });
 
-    it('should set focus state for shadowRoot children', () => {
+    it('should set focus state for shadowRoot children', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        // JSDOM doesn't support :focus-visible
+        this.skip();
+      }
```

Comment:
Maybe we could instead remove the `|jsdom` on L64 if it no longer makes sense? 🤔